README.md

```diff
@@ -50,3 +92,12 @@ Licensed under the GPLv2
     You should have received a copy of the GNU General Public License
     along with this program; if not, write to the Free Software
     Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+
+See the [COPYING][copying] file for the full license information.
+
+[ratbagd-readme]: https://github.com/libratbag/libratbag/blob/master/README.md#running-ratbagd-as-dbus-activated-systemd-service
+[meson]: http://mesonbuild.com/
+[ninja]: https://ninja-build.org/
+[kanban]: https://github.com/orgs/libratbag/projects/1
+[wiki-redesign]: https://github.com/libratbag/piper/wiki/Piper-Redesign
+[copying]: https://github.com/libratbag/piper/blob/master/COPYING
```

Comment:
Genuine question: I know we are not using doxygen for piper, but if we ever, would this links be handled properly?

Also, isn't that simpler to have them directly in the text because when I clone the repo, I read the text through vim^Wmy-favorite-text-editor* and it's easier to have the link just there.

\* not raising any trolls here, though vim is better than anything else